.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
next time please add some description to the PR, including an image or even a gif

Reply:
I see a major bug in the application
![image](https://user-images.githubusercontent.com/9549288/170877135-5a5e2f79-218d-4264-86c9-7111073cbe4a.png)

please make sure it's all working properly before submitting a pull request
let me know when it's fixed so I can start the review